packages/emitter-framework/src/typescript/utils/operation.ts

```diff
@@ -21,21 +22,25 @@ export function getReturnType(
 export interface BuildParameterDescriptorsOptions {
   params?: ts.ParameterDescriptor[] | string[] | undefined;
   mode?: "prepend" | "append" | "replace";
+  suffixRefkey?: Refkey;
```

Comment:
I'm curious about how this is intended to be used - what is a `suffixRefkey` and when would I pass one in?